mappings/net/minecraft/world/BlockView.mapping

```diff
@@ -33,7 +33,7 @@ CLASS net/minecraft/class_1922 net/minecraft/world/BlockView
 		ARG 2 pos
 	METHOD method_32882 (Lnet/minecraft/class_5702;)Lnet/minecraft/class_3965;
 		ARG 0 context
-	METHOD method_35230 getBlockEntityOrEmpty (Lnet/minecraft/class_2338;Lnet/minecraft/class_2591;)Ljava/util/Optional;
+	METHOD method_35230 getBlockEntity (Lnet/minecraft/class_2338;Lnet/minecraft/class_2591;)Ljava/util/Optional;
```

Comment:
Hmm, it says you need to update name in a few other places https://github.com/FabricMC/yarn/pull/2277/checks#step:5:194